src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = (a) => {
```

Comment:
You can directly return it  as an anonymous arrow function. And also you can add 0 as a default value. So if no argument is passed, it would still work.